src/coreclr/jit/objectalloc.cpp

```diff
@@ -3845,13 +3845,21 @@ bool ObjectAllocator::CheckCanClone(CloneInfo* info)
     // (if it does, optimization does not require cloning, as
     // there should be only one reaching def...)
     //
+    // For now we're going to just go ahead and clone, despite the
+    // fact that this is wasteful.
+    //
+    // In the transformation that follows the cloned path is the
+    // "fast" path and gets properly specialized; the original "slow
+    // path" will become unreachable and get cleaned up later on.
+    //
+    // See dotnet/runtime issue #117204.
```

Comment:
Not sure if this is current practice, but the JIT coding guidelines say not to include bug IDs in source code [here](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/clr-jit-coding-conventions.md#713-bug-ids) -- should we get rid of this?

Reply:
Though I suppose if you're going to come back to this soon, we can leave this in for now.